node/network/src/proxy/registry.rs

```diff
@@ -0,0 +1,25 @@
+use std::sync::Arc;
+
+use crate::proxy::{Proxy, ProxyHandler};
+use crate::proxy::debug::DebugHandler;
+use crate::proxy::dropout::DropoutHandler;
+use crate::proxy::throttling::ThrottlingHandler;
+use crate::proxy::benchmark::BenchmarkHandler;
+
+impl Proxy {
```

Comment:
This should be implementing as part of `network.rs` in `config` crate, instead of here.

Reply:
Duplicated, closing this.